File: URI/Haskell/1008.hs
import Text.Printf

main = do
    a <- getLine 
    b <- getLine
    c <- getLine 
    putStrLn $ "NUMBER = " ++ a
    printf "SALARY = U$ %.2f\n" $ salario (read b) (read c)

salario :: Int -> Double -> Double 
salario horas preco = preco * fromIntegral horas

File: URI/Java/src/URI/_1008.java
package URI;

import java.util.Scanner;

public class _1008 {

    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int numero = sc.nextInt();
        int horas = sc.nextInt();
        double valor = sc.nextDouble();
        double salario = valor * horas;
        
        System.out.println("NUMBER = " + numero);
        System.out.printf("SALARY = U$ %.2f\n",salario);
    }
}
